### PR TITLE
5.2 release notes

### PR DESCRIPTION
**Pull request by promanenko**

### CODE REVIEW

**Review by utkukaratas — APPROVED** [review]

**Comment by barisbasaran** [comment]
Are we missing tickets below? 
https://hazelcast.atlassian.net/browse/MC-1687 
https://hazelcast.atlassian.net/browse/MC-1549

File: docs/modules/release-notes/pages/5-2-0.adoc [hunk 1]
```diff
@@ -9,42 +9,56 @@
...
 * SQL Browser: connector wizard for Kafka connector. [MC-1435]
-* SQL Browser: connector wizard for File connector. [MC-1435]
+* SQL Browser: connector wizard for File connector. [MC-1436]
 * Introduced customizable tables, in terms of selected columns and order. [MC-1074]
 * Introduced cluster connection diagnosis functionality. [MC-1332]
 * Added Map Index statistics table. [MC-153]
 * Added a chart for Tiered Storage usage monitoring. [MC-1478]
 * Added support for Command Line Client (CLC). [MC-1548]
+* HealthCheck: added checks for TieredStorage metrics and configuration. [MC-1476][MC-1477][MC-1481]
+* HealthCheck: added checks for number of connected clients. [1581]
```
Comment (Serdaro) [inline docs/modules/release-notes/pages/5-2-0.adoc:19]:
```
* Added health checks for the number of connected clients, and Tiered Storage metrics and configuration. [MC-1476], [MC-1477], [MC-1481], [MC-1581]

**Comment by promanenko** [comment]
> Are we missing tickets below? https://hazelcast.atlassian.net/browse/MC-1687 https://hazelcast.atlassian.net/browse/MC-1549

https://hazelcast.atlassian.net/browse/MC-1549 - this one was caused by the migration to Spring Boot in 5.2. It means customers didn't observe this issue in 5.1 and earlier and we don't inform them that we had some intermediate issues during the development cycle.

https://hazelcast.atlassian.net/browse/MC-1687 - it's related to PhoneHome which is our internal functionality and doesn't affect the users.

In general we don't add to RN tickets:
- related to PH
- specific for Cloud integration
- bugfixes not present in the released artifacts

**Review by Serdaro — COMMENTED** [review]
Minor suggestions.

File: docs/modules/release-notes/pages/5-2-0.adoc [hunk 2]
```diff
@@ -9,42 +9,56 @@
...
 * MC no longer starts RocksDB for metrics persistence on unsupported operating systems (currently z/os). [MC-1486]
 * Added progress indicator for Cluster remove operation. [MC-1471]
 * TTL and Max Idle in Map Browser are shown as Unlimited if set to 0 or more than 1000 years. [MC-1392]
-
+* Client version is shown even if the client stats are disabled. [MC-1611]
+* Introduced a label for client filtering status in the left menu. [MC-1560]
+* Improved CP Session metrics. [MC-1674]
 
 [[fixes-52BETA1]]
 == Fixes
```
[the remark this answers is not in the document]
Comment (promanenko) [reply docs/modules/release-notes/pages/5-2-0.adoc:48]: we have `MC` in the previous RN, I think it's fine since RN are MC-specific

**Review by Serdaro — APPROVED** [review]

**Comment by jgardiner68** [comment]
** You can now get Map Browser information for a record in SQL query results. [MC-1438]
This is specific to SQL queries on maps

**Pull request merged**